testgrid/config/config.yaml

```diff
@@ -4078,6 +4099,11 @@ dashboard_groups:
   - sig-cluster-lifecycle-1.6-upgrade-skew
   - sig-cluster-lifecycle-multi-platform
 
+- name: sig-multicluster
+  dashboard_names:
+  - sig-multicluster-gce
```

Comment:
Should this be ``sig-multicluster-federation-old-gce``?

Reply:
Updated the PR